website/src/env.d.ts

```diff
@@ -1 +1,3 @@
+/// <reference path="../.astro/types.d.ts" />
 /// <reference types="astro/client" />
+declare module 'set-cookie-parser';
```

Comment:
Why is this necessary?

Reply:
When I don't add this I get this error from `tsc`:

```
29 import { parseString } from 'set-cookie-parser'
                               ~~~~~~~~~~~~~~~~~~~
Found 1 error in node_modules/auth-astro/server.ts:29
```
This is inside of one of our imported libraries (auth-astro), and I did not find any other way, that typescript does not check our dependencies as well... Do you know of a way?

Reply:
Fine by me. Probably also doesn't hurt. Maybe add a comment on why the `declare` is necessary?